src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: "Commonly Used Dart Libraries"
+description: "What are some of the most useful and popular Dart libraries and where can you learn more?"
+---
+
+You can use many Dart libraries when writing your code. You can find
+Dart libraries in three ways:
+
+* The core libraries&mdash;such as dart:core, dart:async, and dart:collection&mdash;are
+  distributed with the SDK and documented at [api.dartlang.org]({{site.dart_api}}).
+* Libraries shared with the Dart community are distributed as library packages,
+  published at [pub.dartlang.org](https://pub.dartlang.org/).
+  The [pub](/tools/pub/) tool allows you to create, publish, and manage library packages.
+* You can use libraries from GitHub or your local file system by placing them in the
+  `/lib` directory of your [application's directory
+  structure](https://www.dartlang.org/tools/pub/package-layout#public-directories).
+
+This document discusses the first two kinds of libraries,
+and tells you where to learn more about some of the most widely used Dart libraries.
+
+<aside class="alert alert-info" markdown="1">
+**Tip:**
+If you don't find the functionality you need on [api.dartlang.org]({{site.dart_api}}),
+check [pub.dartlang.org](https://pub.dartlang.org/).
+</aside>
+
+Looking for web, server, or Flutter libraries?
+See [Specialized libraries](#specialized-libraries).
+
+## Dart SDK libraries
+
+The SDK libraries (dart:core, dart:async, dart:math, dart:convert) contain the
```

Comment:
Add "such as" or something else to let them know that these aren't the only SDK libraries.